### DISCUSSION TITLE
alist添加阿里open的资源盘，然后访问时显示没有权限

### DISCUSSION THREAD

**Discussion by SyberRabbit** [AlistGo/alist#7443] [Q&A] [answered]
具体报错如下：
`failed get objs: failed to list objs: UserNotAllowedAccessDrive:user not allowed access drive`
一开始发现资源盘和备份盘无法同时使用，怎么调都只能访问备份盘。
那我只能添加两次阿里open，一个添加备份盘，一个添加资源盘。
现在的情况是，备份盘能用，资源盘就报错
现在过了几天又反过来了，备份盘报错，资源盘能用

**Comment by anwen-anyi** [comment, marked as answer]
看文档里面有写了，被风控禁用了第三方权限

**Reply by sevenjustin** [reply]
知道要怎么解吗？
